perspective.el

```diff
@@ -1176,7 +1184,7 @@ visible in a perspective as windows, they will be saved as
                                   target-file))
                (not (or current-prefix-arg
                         (yes-or-no-p "Target file exists. Overwrite? "))))
-      (error "persp-state-save cancelled"))
+      (error "Persp-state-save cancelled"))
```

Comment:
I disagree with this one since it is a literal function name.

Reply:
OK, revert this change.

Reply:
How about `Cancelled persp-state-save`?